frameworks/helloworld/src/main/dist/normal.yml

```diff
@@ -0,0 +1,15 @@
+# This service will launch tasks that will run as long as indicated by the sleep value.
```

Comment:
This comment is not valid. Also, can one not reuse an existing definition for this?

Reply:
yes this seems to be mostly a repeat of simple.yml, no need to reinvent it